diff --git a/node_modules/solid-js/types/jsx.d.ts b/node_modules/solid-js/types/jsx.d.ts
index 74218aa..31ad8d7 100644
--- a/node_modules/solid-js/types/jsx.d.ts
+++ b/node_modules/solid-js/types/jsx.d.ts
@@ -10,6 +10,7 @@ export namespace JSX {
   type Element =
     | Node
     | ArrayElement
+    | ReadonlyArrayElement
     | FunctionElement
     | string
     | number
@@ -17,6 +18,7 @@ export namespace JSX {
     | null
     | undefined;
   interface ArrayElement extends Array<Element> {}
+  interface ReadonlyArrayElement extends ReadonlyArray<Element> {}
   interface FunctionElement {
     (): Element;
   }
